File: entrega2/menu/src/main/java/com/aps/menu/entidades/Cliente.java
package com.aps.menu.entidades;

import javax.persistence.Entity;

@Entity
public class Cliente extends Conta{
    private String nome;
    private String endereco; // MUDAR PARA CLASSE ENDERECO


    public Cliente(Long id, String email, String senha, String nome, String endereco) {
        super(id, email, senha);
        this.nome = nome;
        this.endereco = endereco;
    }

    public Cliente(Long id, String email, String senha) {
        super(id, email, senha);
    }

    public Cliente () {
        
    }
}

File: rup/menu-app/src/entidades/cliente.ts
import { Conta } from "./conta";
import { Endereco } from "./endereco";

export class Cliente extends Conta {
    private nome: string;
    private endereco: Endereco;

    constructor(id: number, email: string, senha: string, nome: string, endereco: Endereco) {
        super(id, email, senha);
        this.nome = nome;
        this.endereco = endereco;
    }

    public getNome() : string {
        return this.nome;
    }
    
    public getEndereco() : Endereco {
        return this.endereco;
    }
}